rwgame/RWGame.cpp

```diff
@@ -400,17 +403,27 @@ int RWGame::run() {
         }
         RW_PROFILE_END();
 
-        auto now = clock.now();
-        float timer =
-            std::chrono::duration<float>(now - last_clock_time).count();
-        last_clock_time = now;
-        accum += timer * timescale;
+        time_physics_curr = std::chrono::steady_clock::now();
+
+        deltaPhysics = std::chrono::duration<float>(time_physics_curr - time_physics_prev).count();
+
+        world->dynamicsWorld->stepSimulation(deltaPhysics);
+        time_physics_prev = time_physics_curr;
+
+        deltaGame = std::chrono::duration<float>(std::chrono::steady_clock::now() - time_gameclock).count();
 
         RW_PROFILE_BEGIN("Update");
+<<<<<<< HEAD
         if (accum >= GAME_TIMESTEP) {
             if (!StateManager::currentState()) {
+=======
+        while (deltaGame >= GAME_TIMESTEP) {
+            if (StateManager::get().states.size() == 0) {
+>>>>>>> Interpolating dynamic objects
```

Comment:
Merge conflict markers, replacing ``StateManager::get().states.size() == 0`` with ``!StateManager::currentState()`` should be correct.